contributing_to_docs/doc_guidelines.adoc

```diff
@@ -1650,6 +1650,8 @@ Text for admonition
 ====
 ....
 
+See the link:https://redhat-documentation.github.io/supplementary-style-guide/#admonitions[Red Hat Supplementary style guide for the valid admonition types and their defiitions].
```

Comment:
Suggestion:
```suggestion
See the link:https://redhat-documentation.github.io/supplementary-style-guide/#admonitions[Red Hat Supplementary style guide] for the valid admonition types and their definitions.
```
Up to you! I just feel that limiting the link text to the name of the destination is specific enough without making it too long.